src/volatile_memory.rs

```diff
@@ -30,7 +30,6 @@ use std::fmt;
 use std::io::{self, Read, Write};
 use std::marker::PhantomData;
 use std::mem::{align_of, size_of};
-use std::ptr::copy;
```

Comment:
Hmm, on second though, would you mind dropping this commit for now? I think the issue you opened ensures we keep tracking the discussion, and after looking at the performance implications some more I think we might end up replacing `copy_slice` with `copy` or something similar throughout if the difference is significant (subject to performance testing ofc) and after the guaranteed atomicity aspect is no longer a requirement.

Reply:
sure, i will skip it to avoid performance regression.